tests/src/JIT/Performance/CodeQuality/BenchF/Adams/Adams.cs

```diff
@@ -91,16 +108,36 @@ public static void Test()
             }
         }
     }
+#endif
 
-    private static bool TestBase()
+    [MethodImpl(MethodImplOptions.NoOptimization)]
+    public static int Main(string[] argv)
     {
-        bool result = Bench();
-        return result;
-    }
+        if (argv.Length > 0)
```

Comment:
Don't see args in any of benchstone benchmarks. No, I don't think it'll be good to add args parsing machinery here.